language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/helpers/LedgerServices.scala

```diff
@@ -43,7 +43,7 @@ final class LedgerServices(val ledgerId: String) {
 
   import LedgerServices._
 
-  val executionContext: ExecutionContext = global
+  val executionContext: ExecutionContext = global // TODO: is global the right execution context? Usually it's passed implicitly -- why is this val not implicit?
```

Comment:
Implicitly passing values around sounds nice, until you run into the situation where you're trying to figure out where the value comes from.
IMHO, implicit parameters are acceptable for typeclass-"lookup", but anything else should really be passed explicitly. It's more verbose, but also much clearer what's going on.

Reply:
Agreed, implicits can be overused. I do think they can make sense beyond the "type class" use case.  I'd agree without hesitation they should be explicit when you're juggling multiple execution contexts (which seems rare?). Here, I added that comment because the code is using the `global` execution context (and only that one), and this really seems like a "background" decision.

Reply:
Thanks for the input. I've removed the implicit passing of the `ExecutionContext` in this PR.